.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG] Description"
+labels: 'Issue: Bug'
+assignees: ''
+
+---
```

Comment:
FYI. In this issue template, I'll later update it with a issue track list (similar to the PR checklist) to ensure the user provides all the information we need.